src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DeliverySessionServiceTest.kt

```diff
@@ -196,7 +205,38 @@ class DeliverySessionServiceTest @Autowired constructor(
       assertThat(error.message).contains("can't schedule new appointment for session; latest appointment has no feedback delivered")
     }
   }
+  @Test
+  fun `superseded flag set to false when pop attended appointment`() {
```

Comment:
It looks like only `Attended.{YES, NO}` cases are tested. Can we test the `LATE` case?

Reply:
Late is the same as attended?